ProcessCIW/Models/CIW.cs

```diff
@@ -41,6 +41,7 @@ class CIW
         private string _SponsorAlternateEmailAddress2;
         private string _SponsorAlternateEmailAddress3;
         private string _SponsorAlternateEmailAddress4;
+        private string _ApproximiateInvestigationDate;
```

Comment:
Please add comment that this is the prior investigation date.

Reply:
Comment added to identify Column name in database that corresponds with _ApproximiateInvestigationDate.